src/com/sun/jna/Platform.java

```diff
@@ -352,4 +356,30 @@ static String getNativeLibraryResourcePrefix(int osType, String arch, String nam
         }
         return osPrefix;
     }
+
+    public static String getMultiArchPath() {
```

Comment:
I don't see why this was moved into public visibility. I can see why people might want this, but this is not a fix, but a feature.

Reply:
This is quite handy to have this available.